.storybook/webpack.config.js

```diff
@@ -5,13 +5,15 @@ const sharify = require("./sharify")
 
 const ForkTsCheckerNotifierWebpackPlugin = require("fork-ts-checker-notifier-webpack-plugin")
 const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin")
+const SimpleProgressWebpackPlugin = require("simple-progress-webpack-plugin")
 
 const webpack = require("webpack")
 const merge = require("webpack-merge")
-const genDefaultConfig = require("@storybook/react/dist/server/config/defaults/webpack.config.js")
 
 env.load()
 
+const cacheDirectory = path.resolve(process.cwd(), ".cache")
```

Comment:
I think it’s possible to run `yarn start` from a subdirectory, which means this would create a new cache dir, so better to make it relative to `__dirname`

Reply:
Ahh, good point.